content/en/admin/upgrading.md

```diff
@@ -59,14 +59,14 @@ systemctl reload mastodon-web
 The `reload` operation is a zero-downtime restart, also called "phased restart". As such, Mastodon upgrades usually do not require any advance notice to users about planned downtime. In rare cases, you can use the `restart` operation instead, but there will be a (short) felt interruption of service for your users.
 {{< /hint >}}
 
-Rarely, the **streaming API** server is also updated and requires a restart:
```

Comment:
@Gargron my thinking here is that we shouldn't be discouraging people from upgrading the streaming server, but just warn them that it can cause a temporary spike in resource consumption as users reconnect. (though this should be mitigated thanks to the reconnection logic)